frontend_lib/src/helper.js

```diff
@@ -679,37 +678,21 @@ export const scrollIntoViewIfNeeded = (elementToScrollTo, fixedContainer) => {
 export const darkenColor = (c) => color(c).darken(0.15).hex()
 export const lightenColor = (c) => color(c).lighten(0.15).hex()
 
-export const buildContentPathBreadcrumbs = async (apiUrl, content, props) => {
+export const buildContentPathBreadcrumbs = async (apiUrl, content) => {
   const fetchGetContentPath = await handleFetchResult(
     await getContentPath(apiUrl, content.workspace_id, content.content_id)
   )
 
   switch (fetchGetContentPath.apiResponse.status) {
     case 200:
-      GLOBAL_dispatchEvent({
-        type: CUSTOM_EVENT.APPEND_BREADCRUMBS,
-        data: {
-          breadcrumbs: fetchGetContentPath.body.items.map(crumb => ({
-            url: PAGE.WORKSPACE.CONTENT(content.workspace_id, crumb.content_type, crumb.content_id),
-            label: crumb.label,
-            link: (
-              <Link to={PAGE.WORKSPACE.CONTENT(content.workspace_id, crumb.content_type, crumb.content_id)}>
-                {crumb.label}
-              </Link>
-            ),
-            type: BREADCRUMBS_TYPE.APP_FEATURE
-          }))
-        }
-      })
-      break
+      return fetchGetContentPath.body.items.map(crumb => ({
+        link: PAGE.WORKSPACE.CONTENT(content.workspace_id, crumb.content_type, crumb.content_id),
+        label: crumb.label,
+        type: BREADCRUMBS_TYPE.APP_FEATURE,
+        isALink: true
+      }))
     default:
-      GLOBAL_dispatchEvent({
-        type: CUSTOM_EVENT.ADD_FLASH_MSG,
-        data: {
-          msg: props.t('Error while getting breadcrumbs'),
-          type: 'warning',
-          delay: undefined
-        }
-      })
+      console.error('Error getting breadcrumbs data', fetchGetContentPath)
+      throw new Error('')
```

Comment:
Shouldn't the exception also contain the message?

Reply:
It should yes, but since we don't have any exception management, I didn't know what to put in it and it slipped my mind.

My expectations would be to something like an exception code to be handled by a generic process. 
Only consistent possibility here would be to put a generic message like `Breadcrumb exception` ?

Reply:
added the same message as console.error

Reply:
fixed